packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -193,7 +193,7 @@ const NotificationsDialog = ({
             removeNotificationMessageByKey(item.key);
             removeNotificationMessage({
                 key: item.key,
-                should_show_again: false,
+                should_show_again: item.should_show_again || false,
```

Comment:
should_show_again is true for many other notifications. will this change effect their implementation?